network/files/grains_napalm.py

```diff
@@ -0,0 +1,319 @@
+# -*- coding: utf-8 -*-
```

Comment:
This file belongs to _grains directory in the similar matter as modules and states.